.github/workflows/build_test.sh

```diff
@@ -100,6 +101,19 @@ add-apt-repository -y ppa:upstream-systemd-ci/systemd-ci
 apt-get -y update
 apt-get -y build-dep systemd
 apt-get -y install "${PACKAGES[@]}"
+# Install libbpf from Hirsute repos without upgrading the rest of the system
```

Comment:
This is a bit hicky - we have already a PPA with some backports, can't we add it there?

Reply:
It indeed is. I have no idea where the backports PPA repo is or who maintains it (I only know we use it - `ppa:upstream-systemd-ci/systemd-ci`), so if it could be added there, it would be stellar.

Reply:
let me look into that

Reply:
I added you; please let me know if uploading still fails.

Reply:
thank you!

Reply:
@mrc0mmand it's published now, try again after removing this?